workers/docker-worker/release.sh

```diff
@@ -1,30 +1,46 @@
-#!/bin/bash -e
+#! /bin/bash
 
-echo "Releasing docker-worker on Github"
+set -e
 
-if [ "$DOCKER_WORKER_GITHUB_TOKEN" == "" ]; then
-  echo "You need to define the environment variable DOCKER_WORKER_GITHUB_TOKEN" >&2
-  echo "with a valid Github personal token." >&2
-  echo "If you intended to only deploy AMIs, then run the deploy.sh script." >&2
-  exit 1
-fi
+OUTPUT=docker-worker.tgz
+while getopts "o:" opt; do
+    case "${opt}" in
+        o)  OUTPUT=$OPTARG
+            ;;
+    esac
+done
 
-remote=$(git remote -v | grep 'taskcluster/docker-worker' | head -1 | awk '{print $1}')
-if [ "$remote" == "" ]; then
-  git remote add tcdw git@github.com/taskcluster/docker-worker
-  remote=tcdw
-fi
+DW_ROOT=$(mktemp -d)
+trap "rm -rf $DW_ROOT" EXIT
 
-git fetch $remote
+# create an easy-to-use thing that will find the right paths, etc.
+mkdir -p $DW_ROOT/bin
+cat > $DW_ROOT/bin/docker-worker <<'EOF'
+#! /bin/bash
+DW_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd )"
+NODE=$DW_ROOT/node/bin/node
+exec $NODE $DW_ROOT/src/bin/worker.js "${@}"
+EOF
+chmod +x $DW_ROOT/bin/docker-worker
 
-branch=$(git rev-parse --abbrev-ref HEAD)
-current_branch_head_sha=$(git rev-parse $branch)
-remote_branch_head_sha=$(git rev-parse $remote/$branch)
+# install docker-worker itself
+for f in src schemas .npmignore package.json yarn.lock config.yml bin-utils; do
+    cp -r $PWD/$f $DW_ROOT
+done
 
-if [ "$current_branch_head_sha" != "$remote_branch_head_sha" ]; then
-  echo "$branch and $remote/$branch mismatch!" >&2
-  exit 1
-fi
+# Install Node
+NODE_VERSION=12.11.0
+mkdir $DW_ROOT/node
+curl https://nodejs.org/dist/v13.14.0/node-v13.14.0-linux-x64.tar.xz | tar -C $DW_ROOT/node --strip-components=1 -xJf -
```

Comment:
maybe we should say in the docs that this tarball is only for linux x64 and not arm or other platforms